src/regtech_api_commons/oauth2/oauth2_admin.py

```diff
@@ -81,7 +81,12 @@ def upsert_group(self, lei: str, name: str) -> str:
 
     def get_group(self, lei: str) -> Dict[str, Any] | None:
         try:
-            return self._admin.get_group_by_path(f"/{lei}")
+            group = self._admin.get_group_by_path(f"/{lei}")
+            if group and "id" in group:
+                return group
+            else:
+                log.error(f"Unexpected results from get_group_by_path: {group}")
```

Comment:
oh... that's fun... so non-existent group just returns a json result instead of throwing an error now? 😞 :